pandas/core/generic.py

```diff
@@ -862,6 +862,8 @@ def rename(self, *args, **kwargs):
         copy = kwargs.pop('copy', True)
         inplace = kwargs.pop('inplace', False)
         level = kwargs.pop('level', None)
```

Comment:
I *think* there is an issue about explicity listing the signatures in pandas/core/series and pandas/core/frame, for rename. would take that in this PR as well.

Reply:
Sure, happy to do so, just to clarify, you'd like to me to implement them as specific kwargs rather than **kwargs, and default them to None?

Reply:
defaults should be to what they actually default now, these are passed as kwargs to the generic routine, this is mainly to have a nice signature.